ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -995,42 +742,35 @@ class Tab: NSObject {
 
   var displayTitle: String {
     if let displayTabTitle = fetchDisplayTitle(using: url, title: title) {
-      syncTab?.setTitle(displayTabTitle)
       return displayTabTitle
     }
 
     // When picking a display title. Tabs with sessionData are pending a restore so show their old title.
     // To prevent flickering of the display title. If a tab is restoring make sure to use its lastTitle.
-    if let url = self.url, InternalURL(url)?.isAboutHomeURL ?? false, !restoring {
-      syncTab?.setTitle(Strings.Hotkey.newTabTitle)
+    if let url = self.url, InternalURL(url)?.isAboutHomeURL ?? false, !isRestoring {
```

Comment:
so we no longer has the title flickering? if not might as well remove the comments mentioning why we need to set the lastTitle.

Reply:
No change here other than not setting the syncTab title